test/ecto/multi_test.exs

```diff
@@ -496,4 +496,19 @@ defmodule Ecto.MultiTest do
       end
     end
   end
+
+  describe "dynamic repo" do
```

Comment:
Let's please move this to `repo_test`, since we already have some dynamic_repo tests in there. WDYT?